src/CommunicationLayer/Logging/Logging.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+
+#include <QDebug>
+#include <QFile>
+#include <QDate>
+#include <QDir>
+#include <QJsonDocument>
+
+#include "Logging.h"
+
+namespace
+{
+    QString LOG_DIR = "log";
```

Comment:
We've been having issues with the LOG_DIR on other programs, can you verify if this will work if the dashboard is started from outside the build directory?

Reply:
The folder called `log` should be created in whatever folder the executable (and config) file is in. I have tested it by moving the these files to another location. Logging is properly initialized and functional.